harness/determined/estimator/_estimator_trial.py

```diff
@@ -504,13 +522,20 @@ def _init_model(self) -> None:
         self.train_spec = tf.estimator.TrainSpec(
             input_fn=repeating_train_fn, hooks=self.train_hooks
         )
+        steps = (
```

Comment:
praise: nice job catching & fixing the issue in the case where users do specify `steps` as of their EvalSpec.